tests/acceptance/features/cliCommands/authAppToken.feature

```diff
@@ -0,0 +1,18 @@
+@env-config
+Feature: create auth token
+  As an admin
+  I want to create App Tokens
+  So that I can use 3rd party apps
+
+  Background:
+    Given the following configs have been set:
+      | config                | value    |
+      | OCIS_ADD_RUN_SERVICES | auth-app |
+      | PROXY_ENABLE_APP_AUTH | true     |
+    And user "Alice" has been created with default attributes
```

Comment:
Are you going to add more tests scenario here in up-coming PRs. If yes, then use of background is ok, but if only one scenario, then better not using background